Metrizable + realcompact => non-measurable cardinality, and improved definition of non-measurable cardinality (#1143)
pi-base/data: spaces/S001103/properties/P000164.md

value: true
---

> The space has cardinality $\omega^{2^\mathfrak{c}}$, which is smaller than any measurable cardinal, if one exists

Reviewer: ```suggestion
The space has cardinality $\omega^{2^\mathfrak{c}}$, which is smaller than every measurable cardinal.
```
